config/templates/terms-of-service.md

```diff
@@ -83,13 +83,24 @@ that you submit to the Instance, you may expose yourself to liability from third
 parties if you post or share such Content without all necessary rights.
 
 You retain all ownership rights you have in the Content that you submit to the
-Instance, but you grant us a limited, non-exclusive, irrevocable, transferable,
-royalty-free, perpetual license to use, copy, store, display, share, distribute,
-communicate and transfer the Content in ways that are consistent with your use
-of the Instance. To the fullest extent possible, you agree to waive or promise
-not to assert against the Administrator all moral rights you may have in the
-Content to the extent those rights are necessary for the Administrator to host
-the Content on the Instance.
+Instance, but you grant us a limited, non-exclusive, transferable, royalty-free,
```

Comment:
Suggest removing the comma at the end of this line (after "royalty-free").

Also suggest removing the arbitration clause altogether, although that may be outside the scope of this PR.

Otherwise, this looks good to me.

Reply:
Yeah, I agree on removing the arbitration clause, or making it optional, but I think that's out of scope of this PR; I was planning on doing another PR for that one.